### PR TITLE
[FIX] account: account group with branches

### PR DESCRIPTION
**Pull request by malb-odoo**
Before this commit when being in a branches environment, creating an account group on the main company was not propagated to the account of the child companies

opw: 4055582

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by Megaaaaaa — APPROVED** [review]
LGTM, thx! :+1:

Since oco is on holidays for three weeks and this is a bugfix, i won't let you hanging, i'll spin the r+ wheel and randomly assign someone else for final review :smile:

File: addons/account/models/account_account.py [hunk 1]
```diff
@@ -901,7 +901,12 @@ def _adapt_accounts_for_account_groups(self, account_ids=None, company=None):
             company_ids = account_ids.company_id.root_id.ids
             account_ids = account_ids.ids
         else:
-            company_ids = self.company_id.ids
+            company_ids = []
+            for company in self.company_id:
+                company_ids.append(company.id)
+                if company.all_child_ids:
+                    company_ids.extend(company.all_child_ids.ids)
```
Comment (epictete) [inline addons/account/models/account_account.py:908]:
```suggestion
            company_ids = self.company_id._accessible_branches()
```
Couldn't we simply do this?

Comment (malb-odoo) [reply addons/account/models/account_account.py]: works perfectly with that :smile:

**Review by epictete — APPROVED** [review]
Thanks for the update :ok_hand: 

@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**